test/source/tests/tests/compose.ts

```diff
@@ -16,6 +16,22 @@ export const defineComposeTests = (testVariant: TestVariant, testWithNewBrowser:
 
   if (testVariant !== 'CONSUMER-LIVE-GMAIL') {
 
+    ava.default.only('compose[global:compatibility] - toggle minimized state by clicking compose window header', testWithNewBrowser(async (t, browser) => {
```

Comment:
Common oversight - please don't leave `.only` checked in - would make our CI skip all tests except yours

Reply:
:man_facepalming: oops, sorry!